### PR TITLE
Feature/search house

### LINKED ISSUE #13: Create index page
List all rooms with name and price, and search with position

### PR DESCRIPTION
**Pull request by lyuboxin**
Resolves #13, create an index page with google map marker.

### CODE REVIEW

File: addroom.html [hunk 1]
```diff
@@ -147,6 +169,9 @@ <h3>
         var cityname = document.getElementById("City").value;
         var provincename = document.getElementById("Provinces").value;
         var postalcode = document.getElementById("code").value;
+        var bed_num = document.getElementById("beds").value;
+        var startdate = document.getElementById("date1").value;
```
Comment (xingweitian) [inline addroom.html]: bad id

File: addroom.html [hunk 2]
```diff
@@ -147,6 +169,9 @@ <h3>
...
+        var enddate = document.getElementById("date2").value;
```
Comment (xingweitian) [inline addroom.html]: bad id

File: addroom.html [hunk 3]
```diff
@@ -147,6 +169,9 @@ <h3>
         var cityname = document.getElementById("City").value;
         var provincename = document.getElementById("Provinces").value;
         var postalcode = document.getElementById("code").value;
```
Comment (xingweitian) [inline addroom.html:171]: please be case sensitive.

for example, City, Provinces, Code, Beds...

File: addroom.html (hunk 1 above)
Comment (xingweitian) [reply addroom.html]: please be more specific.

for example, date_begin, date_end, etc..

File: homepage.html [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+                id="beds"
+                placeholder="1"
+                onchange="remap()"
+              />
+            </div>
+            <div class="form-group">
+              <label for="exampleInputPassword1">
+                Check in
+              </label>
+              <input type="date" class="form-control" id="date1" />
```
Comment (xingweitian) [inline homepage.html]: the same as above. please try to use a more specific id.

File: homepage.html [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+            </div>
+            <div class="form-group">
+              <label for="exampleInputPassword1">
+                Check out
+              </label>
+              <input type="date" class="form-control" id="date2" />
```
Comment (xingweitian) [inline homepage.html]: the same as above. please try to use a more specific id.

File: homepage.html [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+
+      search_form.onsubmit = function(e) {
+        e.preventDefault();
+        document.getElementById("show").innerHTML = "";
+        var title1 = document.createElement("span");
+        title1.id = "sentence";
+        title1.innerHTML = "Search result:";
+        $(".show").append(title1);
+        var addr = document.getElementById("address").value;
+        var begindate = document.getElementById("date1").value;
```
Comment (xingweitian) [inline homepage.html]: please follow the convention of naming of JavaScript.

this link may be useful: [variable naming principles](https://www.w3cplus.com/javascript/variable-naming-principles.html)

**Review by xingweitian — CHANGES_REQUESTED** [review]

**Comment by zhangjiangqige** [comment]
Has this been updated to the cs server? The page there doesn't seem to work for me

**Review by zhangjiangqige — DISMISSED** [review]
make distance an argument in the future

**Review by xingweitian — APPROVED** [review]

**Pull request merged**